addon.xml

```diff
@@ -12,7 +12,7 @@
     <import addon="script.module.tzlocal" version="2.0.0"/>
     <import addon="script.module.future" version="0.17.1"/>
     <import addon="script.module.kodi-six" version="0.0.4"/>
-    <import addon="resource.images.catchuptvandmore" version="1.0.0"/>
+    <import addon="resource.images.catchuptvandmore" version="1.0.23"/>
```

Comment:
Planned version based on ongoing activities. 1.0.23 is the one expected after the merge of https://github.com/xbmc/repo-resources/pull/414

Reply:
Restored to 1.0.0 based on Joao's review. See other comments in this PR.